dashboard/test/models/lti_integration_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class LtiIntegrationTest < ActiveSupport::TestCase
```

Comment:
I am incredibly pro-test person, so this feedback isn't coming from a place of "we don't need no stinkin' tests!" :wink: In fact, this looks test-driven (:heart_eyes:) based on the order of your tests. However, they seem to be coming close to testing Rails vs testing our business logic. Are these tests useful?

The business logic here is: There are 7 required fields for an LtiIntegration object. Would it be cleaner to refactor these to specifically check validity in a single test case?

```ruby
test "should validate required fields" do
    assert_not build(:lti_integration, platform_id: nil).valid? "Missing platform_id should be invalid"
    assert_not build(:lti_integration, issuer: nil).valid? "Missing issuer should be invalid"
    # ...
end
```

My general nit here is that it looks like we're testing a lot, when really we aren't.

Reply:
Thanks for the feedback. I agree, these tests should be slimmed down as they don't provide real value.